commercetools-models/src/main/java/io/sphere/sdk/extensions/Trigger.java

```diff
@@ -13,7 +13,7 @@
 public interface Trigger {
 
 
-    String getResourceTypeId();
+    ExtendedResourceType getResourceTypeId();
```

Comment:
We should document this change in our release notes, just in case that someome used it already 😄

Reply:
Yep i ve done it 👍

Reply:
I feel like the name is a bit misleading. "ExtendedResourceType" sounds like the resource type has been extended with some properties. Simply "ExtensionResourceType" indicates that it's the resource type for extension. WDYT?

Reply:
Sorry, looks like I didn't see it.

Reply:
@lauraluiz i too didn't find a good name, but yours sound good, i will change it 👍